packages/form-builder/src/components/sidebar/primary.js

```diff
@@ -30,14 +28,16 @@ const tabs = [
         content: () => (
             <>
                 <BlockCard
-                    icon={ settings }
+                    icon={settings}
                     title="Form Settings"
                     description="These settings affect how your form functions and is presented, as well as the form page."
                 />
-                <FormSummarySettings />
-                <DonationGoalSettings />
-                <OfflineDonationsSettings />
-                <FormFields />
+                <FormSummarySettings/>
+                <DonationGoalSettings/>
+                <ReceiptSettings/>
+                {/*The settings below have not been implemented yet.*/}
```

Comment:
Ah, ok, I see what you were talking about with the settings that haven't been implemented.